templates/tfengine/recipes/iam_members.hcl

```diff
@@ -56,6 +57,15 @@ schema = {
                   }
                 }
               }
+              project = {
+                description = <<EOF
+                  Project ID to add the IAM policies/bindings
```

Comment:
ID of the project where the resources (e.g. service accounts) belong.

Reply:
took the description from [https://github.com/terraform-google-modules/terraform-google-iam/blob/master/modules/service_accounts_iam/variables.tf](url)

I'll update it based on your suggestion